src/components/EntityInfo.vue

```diff
@@ -46,50 +46,48 @@
               <span class="font-14 mx-3">{{getReasonText || 'Unknown Reason'}}</span>
             </span>
 
-            <template v-else>
-              <!-- View and Change Company Information -->
-              <span v-if="isAllowed(AllowableActions.VIEW_CHANGE_COMPANY_INFO)">
-                <v-btn
-                  small text color="primary"
-                  id="company-information-button"
-                  :disabled="hasBlocker"
-                  @click="promptChangeCompanyInfo()"
-                >
-                  <v-icon medium>mdi-file-document-edit-outline</v-icon>
-                  <span class="font-13 ml-1">View and Change Business Information</span>
-                </v-btn>
+            <!-- View and Change Business Information -->
+            <span v-if="!!businessId && !isHistorical">
```

Comment:
I tweaked the logic a bit here to make more sense:
- don't show unless this is a business (not a temporary registration)
- don't show if business is historical
- disable if not allowed

Same below for Dissolve Business.